File: SmartReceipts/Reports/Generators/ReportCSVGenerator.m
//
//  ReportCSVGenerator.m
//  SmartReceipts
//
//  Created by Jaanus Siim on 25/04/15.
//  Copyright (c) 2015 Will Baumann. All rights reserved.
//

#import "ReportCSVGenerator.h"
#import "Database.h"
#import "Database+CSVColumns.h"

@implementation ReportCSVGenerator

- (NSArray *)receiptColumns {
    return [[Database sharedInstance] allCSVColumns];
}

@end

File: SmartReceipts/Reports/Generators/ReportCSVGenerator.swift
//
//  ReportCSVGenerator.swift
//  SmartReceipts
//
//  Created by Bogdan Evsenev on 03/12/2017.
//  Copyright © 2017 Will Baumann. All rights reserved.
//

class ReportCSVGenerator: ReportGenerator {
    override func receiptColumns() -> [ReceiptColumn] {
        return database.allCSVColumns() as! [ReceiptColumn]
    }
}
